get_ns.py

```diff
@@ -10,14 +10,24 @@ def is_active_line(line):
     """
     return any(ts in line.decode() for ts in TEST_STRINGS)
 
+def is_active_ns(line):
+    return (ts in line.decode() for ts in TEST_STRINGS)
 
 def is_active_domain(f):
     return any(is_active_line(line) for line in open(f, 'rb'))
 
+def is_ns_in_domain(f):
+    return (is_active_ns(line) for line in open(f, 'rb'))
+
 
 directory = [os.path.join(path, f) for f in os.listdir(path)]
 domainfiles = [f for f in directory if os.path.isfile(f)]
-inactive_domains = [f for f in domainfiles if not is_active_domain(f)]
-
+inactive_domains = [f.replace( path+'/', '') for f in domainfiles if not is_active_domain(f)]
```

Comment:
You shouldn't be doing file path surgery by string replacements because it won't work on the PC.

To be honest, I don't see that you need to do this at all - you should just keep track of the name of the domain during the computations.

I'll show you a commit!